Resources/ServerInfo/_CP14/Guidebook_EN/JobsTabs/Innkeepers.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+  # Innkeeping
+  The innkeeper is a role that primarily stays within the town. The responsibilities of an innkeeper include gardening, bartending, and cooking.
```

Comment:
And also spread rumors, give the order to "bring meat and wood", remove corpses in front of the tavern and keep it clean with a mop, bucket and drinking water from a barrel.